draft-ietf-dnsop-svcb-https.md

```diff
@@ -1589,6 +1589,25 @@ two `list-value`s:
     part1
     part2,part3
 
+# HTTP Mapping Summary
+
+This table serves as a non-normative summary of the HTTP mapping for SVCB.
+Future protocol mappings may provide a similar summary table.
+
+|                                  |                                          |
+|----------------------------------+------------------------------------------|
+| **Mapped scheme**                | "https"                                  |
+| **RR type**                      | HTTPS                                    |
+| **Name prefixing**               | None for port 443, otherwise Port-Prefix |
+| **Automatically Mandatory Keys** | `port`, `alpn`, `no-default-alpn`        |
+| **SvcParam defaults**            | `alpn`: \["http/1.1"\]                   |
+| **Special behaviors**            | HTTP to HTTPS upgrade                    |
+|----------------------------------+------------------------------------------|
+
+This table does not indicate any SvcParamKeys that servers are required to
+publish, or that clients are required to implement, because there are none
+in this mapping.
+
```

Comment:
* What happens if a client specifies no-default-alpn but doesn't include an alpn key?
* Aren't clients required to implement all of the automatically-mandatory keys?  Or just that if a server sends them but a client doesn't implement then the record is pruned?  Does this mean alpn is effectively required for clients to implement?

Reply:
> > What happens if a client specifies no-default-alpn but doesn't include an alpn key?
> 
> Assuming s/client/origin, that record is null: its set of advertised ALPNs is empty, so no client will ever attempt to use it.
> 

For consistency with all the other rules for calling non-sensible stuff invalid, should this be made into a more formal "don't do that"? And should clients consider it malformed (at least those clients that can handle "alpn" if you remove it from the mandatory keys)?

Maybe: 'If "no-default-alpn" is indictated, "alpn" MUST be included.  If a clients that recognizes "alpn" SvcParamKeys receives an RR with "no-default-alpn" but no "alpn", the RR must be considered malformed.'

Reply:
This state does violate the "alpn list is one or more entries" scenario, but it would be good to call it out explicitly somewhere, but likely elsewhere from this table.  (If no-default-alpn is specified without an alpn, the RR should be discarded seems like a reasonable behavior.)

Reply:
Opened issue #283

Reply:
And split out PR #284